src/otp/dto/otp.dto.ts

```diff
@@ -4,9 +4,9 @@ import {IsString} from 'class-validator'
 export class OtpDTO {
     @ApiProperty()
     @IsString()
-    token: String
+    token: string
```

Comment:
if i understand correctly this token is jwt right? If yes, it should be in dto

Reply:
@Awesomeman007 Could you clarify this? I'm not sure either.